lib/middleware.py

```diff
@@ -229,17 +232,30 @@ def rbac_group_id_check(rbac_filter: dict, requested_ids: set) -> None:
             abort(HTTPStatus.FORBIDDEN, f"You do not have access to the the following groups: {joined_ids}")
 
 
-def get_rbac_default_workspace() -> UUID | None:
+# TODO: Remove this once no longer testing
+def _temp_add_org_admin_user_identity(identity_header: str) -> str:
+    identity = from_auth_header(identity_header)
+    if not hasattr(identity, "user"):
+        return identity_header
+    else:
+        identity.user["is_org_id"] = True
```

Comment:
Should it be `is_org_admin` inside the dict?

```suggestion
        identity.user["is_org_admin"] = True
```

Reply:
Yes!! I must have had a brain fart, thank you :laughing: